cmd/vic-machine/create/create.go

```diff
@@ -986,14 +986,18 @@ func (c *Create) loadCertificates() ([]byte, *certificate.KeyPair, error) {
 		return certs, nil, err
 	}
 
-	// We just do a direct equality check here - trying to be clever is liable to lead to hard
-	// to diagnose errors
-	if cert.Leaf.Subject.CommonName != c.cname {
-		log.Errorf("Provided cname does not match that in existing server certificate: %s", cert.Leaf.Subject.CommonName)
-		if c.Debug.Debug > 2 {
-			log.Debugf("Certificate does not match provided cname: %#+v", cert.Leaf)
+	if cert.Leaf == nil {
+		log.Warnf("Failed to load x509 leaf: Unable to confirm server certificate cname matches provided cname %q. Continuing...", c.cname)
```

Comment:
If the Leaf is empty, can we do other checks here?
If the common name does not match, we'll return failure, but if we cannot verify, just continue. Feels a bit wrong here.